meta-test-family.spec

```diff
@@ -1,13 +1,13 @@
 %global framework_name moduleframework
 
-Name:           modularity-testing-framework
+Name:           meta-test-family
```

Comment:
There are two options how to handle it:
1) keep the same name and add to spec file http://pkgs.fedoraproject.org/cgit/rpms/coreutils.git/tree/coreutils.spec?h=f26#n130
In our case: Provides: meta-test-family = %{version} - %{release}
2) Mark modularity-testing-framework as obsolete and introduce the new name. I don't know in my mind, If the new package request will be needed. Including new Fedora dist-git repo.